lib/dap/filter/http.rb

```diff
@@ -15,7 +15,7 @@ def extract_elements(data)
     data.
       to_s.
       encode('UTF-8', invalid: :replace, undef: :replace, replace: '').
-      scan(/<([^>]+)>/m).each do |e|
+      scan(/<([^<>]{1,4096})>/m).each do |e|
```

Comment:
Technically `{1,049}` isn't needed to solve the immediate problem. I've included it here to put an upper limit on how long the regex engine will spend on a particular string in the event that is run across data constructed in a particular way.  4096 *should* be enough for us to extract links in real world situations.